percona/controller/pgcluster/controller.go

```diff
@@ -549,6 +615,33 @@ func (r *PGClusterReconciler) reconcileCustomExtensions(cr *v2.PerconaPGCluster)
 		))
 		set.VolumeMounts = append(set.VolumeMounts, extensions.ExtensionVolumeMounts(cr.Spec.PostgresVersion)...)
 	}
+	return nil
+}
+
+func DisableCustomExtensionsInPostgreSQL(ctx context.Context, exec postgres.Executor, customExtensionsForDeletion []string) error {
```

Comment:
I would name this simply as `DisableCustomExtensions` or `DisableCustomExtensionsInDB`, it is unnecessary to mention Postgres since there is nothing other than postgres, and, the name is shorter.